### PR TITLE
feat: add manHour to iteration API

### PR DESCRIPTION
**Pull request by dspo**
#### What this PR does / why we need it:

feat: add manHour to iteration API

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | feat: add manHour to iteration API |
| 🇨🇳 中文    | 为迭代接口增加 manHour 信息 |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .erda/migrations/cmdb/20230522-interation-man-hour.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `dice_iterations`
+    ADD `man_hour` text COMMENT '迭代工时信息';
```
Comment (sfwn) [inline .erda/migrations/cmdb/20230522-interation-man-hour.sql:2]: after `xxx` field.

File: apistructs/issue.go [hunk 2]
```diff
@@ -516,6 +516,11 @@ func (imh *IssueManHour) Convert2String() string {
 	return string(mh)
 }
 
+func (imh *IssueManHour) FromString(s string) *IssueManHour {
+	_ = json.Unmarshal([]byte(s), imh)
```
Comment (sfwn) [inline apistructs/issue.go:520]: log for error

**Review by sfwn — CHANGES_REQUESTED** [review]

File: .erda/migrations/cmdb/20230522-interation-man-hour.sql (hunk 1 above)
Comment (dspo) [reply .erda/migrations/cmdb/20230522-interation-man-hour.sql:2]: need not to resolve

**Review by sfwn — APPROVED** [review]

**Pull request merged**